cloud_functions/api_router/Dockerfile

```
@@ -0,0 +1,25 @@
+COPY requirements.txt .
+RUN pip install --no-cache-dir -r requirements.txt
+
+# Copy all files
+COPY . .
+
+# Expose the Functions Framework port
+EXPOSE 8080
+
+# Launch the HTTP function
+CMD ["functions-framework",
+     "--target", "main.api_router",
```

Hadolint (2.12.0)

[error] unexpected '"'
expecting '#', '\', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, at least one space, or end of input

(DL1000)